### PR TITLE
[DNM] Blessed turfs are visible to people who care about blessed turfs

### PR DESCRIPTION
**Pull request by coiax**
- Chaplains, revenants, anyone with an ethereal jaunt spell will be able to see a glowing yellow shimmer on turfs that are "blessed".
- NOJAUNT_1 still exists as a flag, but that can no longer be set by any players, it's a pure admin only flag, for technical turfs.

- [ ] Sprites for the blessing, tuned so they're visible but not overwhelming
- [ ] Actually making it so only those groups of people (+observers) can see the blessings

### CODE REVIEW

**Comment by coiax** [comment]
Drones are cool.

**Comment by ACCount12** [comment]
I like the idea of blessed tiles being a silent trap more.

**Comment by Buggy123** [comment]
I mean I'd agree with you if it wasn't for the fact you can bless massive areas really quickly via a fire extinguisher.

**Comment by ExcessiveUseOfCobblestone** [comment]
> I mean I'd agree with you if it wasn't for the fact you can bless massive areas really quickly via a fire extinguisher.

I'd agree with you if those spells weren't temporary noclip. Ofc the area is going to be massive when you have to cover every nook and cranny or else it will be evaded.

File: code/modules/mob/mob_movement.dm [hunk 1]
```diff
@@ -212,9 +212,13 @@
 						R.stun(20)
 					return
 				if(stepTurf.flags_1 & NOJAUNT_1)
-					to_chat(L, "<span class='warning'>Holy energies block your path.</span>")
-				else
-					L.loc = get_step(L, direct)
+					to_chat(L, "<span class='warning'>Some strange aura is blocking the way.</span>")
+					return
+				for(var/obj/effect/blessing/B in stepTurf)
```
Comment (MoreRobustThanYou) [inline code/modules/mob/mob_movement.dm]: you can change this to ```if (/obj/effect/blessing in stepTurf)```

File: code/modules/spells/spell_types/ethereal_jaunt.dm [hunk 2]
```diff
@@ -84,12 +84,18 @@
...
-		forceMove(newLoc)
-	else
-		to_chat(user, "<span class='warning'>Some strange aura is blocking the way!</span>")
+
 	movedelay = world.time + movespeed
 
+	if(newLoc.flags_1 & NOJAUNT_1)
+		to_chat(user, "<span class='warning'>Some strange aura is blocking the way.</span>")
+		return
+	for(var/obj/effect/blessing/B in newLoc)
```
Comment (MoreRobustThanYou) [inline code/modules/spells/spell_types/ethereal_jaunt.dm]: you can change this to ```if (/obj/effect/blessing in stepTurf)```

**Review by ghost — CHANGES_REQUESTED** [review]

**Pull request closed without merging**